tests/lib/openstack_query/runners/test_server_runner.py

```diff
@@ -27,134 +27,124 @@ def setUp(self):
         self.instance = ServerRunner(connection_cls=self.mocked_connection)
         self.conn = self.mocked_connection.return_value.__enter__.return_value
 
-    def test_get_projects_get_all(self):
+    def test_parse_meta_params_with_from_projects(self):
         """
-        Tests _get_projects method works expectedly
-        method should return all projects when project param empty
-        """
-        self.conn.identity.projects.return_value = ["project1", "project2"]
-        res = self.instance._get_projects(self.conn)
-        self.conn.identity.projects.assert_called_once()
-        self.assertEqual(res, ["project1", "project2"])
-
-    def test_get_projects_with_valid_project_identifiers(self):
-        """
-        Tests _get_project method works expectedly - with valid project identifiers
+        Tests _parse_meta_params method works expectedly - with valid from_project argument
```

Comment:
Is this unit test for testing we get lists of servers based on a project given?

Reply:
yes - we're testing that `parse_meta_params` works when giving valid project identifiers